static/install

```diff
@@ -9,9 +9,10 @@ source ${SOURCE_DIR}/base/rc/config
 
 apt-get update
 apt-get install -y --no-install-recommends software-properties-common
+
 add-apt-repository -y ppa:nginx/stable
 apt-get update
-apt-get install -y --no-install-recommends nginx
+apt-get install -y --no-install-recommends nginx-extras
```

Comment:
just checking, but does this also pull nginx-full? Or do we only need nginx-common and nginx-extras?